internal/storage/rocksdb/store.go

```diff
@@ -342,34 +343,30 @@ func (rdb *rocksDB) CompareAndSet(key, expect, update []byte) (bool, error) {
 	return err == nil, err
 }
 
-const tempFilePrefix = "rocksdb-sstfile-"
-
-func (rdb *rocksDB) GetSnapshot() ([]byte, error) {
-	defer rdb.opts.statsCli.Timing("rocksdb.snapshot.get.latency.ms", time.Now())
-	snap := rdb.db.NewSnapshot()
-	defer rdb.db.ReleaseSnapshot(snap)
+const sstPrefix = "rocksdb-sstfile-"
 
+func (rdb *rocksDB) generateSST(snap *gorocksdb.Snapshot, cf *gorocksdb.ColumnFamilyHandle, sstDir string) (string, error) {
+	var fileName string
 	envOpts := gorocksdb.NewDefaultEnvOptions()
 	opts := gorocksdb.NewDefaultOptions()
 	sstWrtr := gorocksdb.NewSSTFileWriter(envOpts, opts)
 	defer sstWrtr.Destroy()
 
-	sstFile, err := storage.CreateTempFile(rdb.opts.sstDirectory, tempFilePrefix)
-	if err != nil {
-		return nil, err
+	if fileName = sstDir + "/default.cf"; cf == rdb.ttlCF {
```

Comment:
Check if this hard-coding can be removed in preference for temp file (like the Badger impl.) based approach.

Reply:
default.cf & ttl.cf are two named files which are generated inside the tmp directory which is created during the rocksdb snapshot process. Have removed from hard-coding and moved to constants. If we are able to have a single snapshot file for multi-CF then, we can get rid of these.